src/target/riscv/riscv-013.c

```diff
@@ -2190,9 +2190,7 @@ static unsigned riscv013_data_bits(struct target *target)
 			 * take those into account as well.  For now we assume abstract commands
 			 * support XLEN-wide accesses. */
 			return riscv_xlen(target);
-		} else if (method == RISCV_MEM_ACCESS_UNSPECIFIED)
-			/* No further mem access method to try. */
-			break;
+		}
```

Comment:
An optional detail (if you like):

```suggestion
		} else {
			assert(false);
		}
```

Reply:
Addressed